frontend/src/component/admin/groups/EditGroup/EditGroup.tsx

```diff
@@ -46,18 +45,13 @@ export const EditGroup = ({
 }: IEditGroupProps) => {
     const { refetchGroups } = useGroups();
     const { setToastData, setToastApiError } = useToast();
-    const { uiConfig, isEnterprise } = useUiConfig();
+    const { uiConfig } = useUiConfig();
     const navigate = useNavigate();
 
     const {
-        settings: { enabled: scimSettingEnabled },
+        settings: { enabled: scimEnabled },
     } = useScimSettings();
-    const scimFlagEnabled = useUiFlag('scimApi');
-    const scimEnabled =
-        isEnterprise() &&
-        scimSettingEnabled &&
-        scimFlagEnabled &&
-        Boolean(group?.scimId);
```

Comment:
Ok so the `useScimSettings().enabled` already covers this making it redundant

Reply:
Correct, that's what I'm thinking 👍